packages/expo-updates/e2e/setup/create-updates-test.js

```diff
@@ -22,7 +22,7 @@ function transformAppJson(appJson, projectName, runtimeVersion) {
       ...appJson.expo,
       name: projectName,
       runtimeVersion,
-      jsEngine: 'jsc',
+      // jsEngine: 'jsc',
```

Comment:
revert?

Reply:
Actually we should probably move to hermes by default for testing, as JSC probably won't support all features in future. I'll remove the comment.